srcpkgs/SystemPreferences/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'SystemPreferences'
+pkgname=SystemPreferences
+version=1.1.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="gnustep-make gcc-objc"
+makedepends="gnustep-base-devel gnustep-gui-devel"
+depends="gnustep-base gnustep-gui gnustep-back"
+short_desc="GNUstep preferences application"
+maintainer="Phosu Parsons <founderofjp@gmail.com>"
+license="GPL-2.0-only"
+homepage="http://gnustep.org/experience/systempreferences.html"
+distfiles="ftp://ftp.gnustep.org/pub/gnustep/usr-apps/${pkgname}-${version}.tar.gz"
+checksum=c7dffa3b33949e26c319aa2adbda8e9168e51d2f1a0790d5ec0d609a915ec8e0
+nocross="objc doesn't get cross build at present within gcc (see gnustep-base)"
+shlib_provides="libPreferencePanes.so.1"
```

Comment:
You don't need this. You can ignore the warning for the library unless something else needs to link to it.

Reply:
I've been working on [gworkspace](https://github.com/phossil/void-packages/blob/gworkspace/srcpkgs/gworkspace/template) in the meantime and it might be necessary because the library is mentioned in the build log.
```shell
Making all in mdfind ...
Making all for tool mdfind...
 Compiling file mdfind.m ...
 Linking tool mdfind ...
Making all in Preferences ...
Making all for bundle MDIndexing...
 Creating MDIndexing.prefPane/....
 Compiling file MDIndexing.m ...
 Compiling file CategoriesEditor.m ...
 Compiling file CategoryView.m ...
 Compiling file StartAppWin.m ...
In file included from MDIndexing.m:26:
MDIndexing.h:31:11: fatal error: PreferencePanes/PreferencePanes.h: No such file or directory
   31 |   #import <PreferencePanes/PreferencePanes.h>
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
distcc[6062] ERROR: compile MDIndexing.m on localhost failed
In file included from CategoriesEditor.m:28:
MDIndexing.h:31:11: fatal error: PreferencePanes/PreferencePanes.h: No such file or directory
   31 |   #import <PreferencePanes/PreferencePanes.h>
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[5]: *** [/usr/share/GNUstep/Makefiles/rules.make:515: obj/MDIndexing.obj/MDIndexing.m.o] Error 1
make[5]: *** Waiting for unfinished jobs....
distcc[6066] ERROR: compile CategoriesEditor.m on localhost failed
make[5]: *** [/usr/share/GNUstep/Makefiles/rules.make:515: obj/MDIndexing.obj/CategoriesEditor.m.o] Error 1
make[4]: *** [/usr/share/GNUstep/Makefiles/Instance/bundle.make:193: internal-bundle-run-compile-submake] Error 2
make[3]: *** [/usr/share/GNUstep/Makefiles/Master/rules.make:297: MDIndexing.all.bundle.variables] Error 2
make[2]: *** [/usr/share/GNUstep/Makefiles/Master/bundle.make:37: internal-all] Error 2
make[1]: *** [/usr/share/GNUstep/Makefiles/Master/serial-subdirectories.make:53: internal-all] Error 2
make: *** [/usr/share/GNUstep/Makefiles/Master/serial-subdirectories.make:53: internal-all] Error 2
=> ERROR: gworkspace-0.9.4_1: do_build: '${make_cmd} ${makejobs} ${make_build_args} ${make_build_target}' exited with 2
=> ERROR:   in do_build() at common/build-style/gnu-configure.sh:13
phossil in void-packages on  gworkspace [!] took 3m22s ✗ 
```

Reply:
Terminal output when attempting to build gworkspace with SystemPreferences and PDFKit:
```shell
   Stripped position-independent executable: /usr/bin/mdextractor
   Stripped position-independent executable: /usr/bin/mdfind
=> gworkspace-0.9.4_1: running post-install hook: 10-pkglint-devel-paths ...
=> gworkspace-0.9.4_1: running post-install hook: 11-pkglint-elf-in-usrshare ...
=> gworkspace-0.9.4_1: running post-install hook: 12-rename-python3-c-bindings ...
=> gworkspace-0.9.4_1: running post-install hook: 13-pkg-config-clean-xbps-cross-base-ref ...
=> gworkspace-0.9.4_1: running post-install hook: 99-pkglint-warn-cross-cruft ...
=> gworkspace-0.9.4_1: running pre-pkg hook: 03-rewrite-python-shebang ...
=> gworkspace-0.9.4_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libc.so.6 <-> glibc>=2.29_1
   SONAME: libsqlite3.so.0 <-> sqlite>=3.31.1_3
   SONAME: libFSNode.so.0 <-> gworkspace (ignored)
   SONAME: libDBKit.so.0 <-> gworkspace (ignored)
   SONAME: libgnustep-gui.so.0.28 <-> gnustep-gui>=0.28.0_1
   SONAME: libgnustep-base.so.1.27 <-> gnustep-base>=1.27.0_1
   SONAME: libobjc.so.4 <-> libobjc>=4.7.3_12
   SONAME: libm.so.6 <-> glibc>=2.29_1
   SONAME: libPreferencePanes.so.1 <-> UNKNOWN PKG PLEASE FIX!
   SONAME: libInspector.so.0 <-> gworkspace (ignored)
   SONAME: libOperation.so.1 <-> gworkspace (ignored)
   SONAME: libMDKit.so.0 <-> gworkspace (ignored)
   SONAME: libpthread.so.0 <-> glibc>=2.29_1
=> ERROR: gworkspace-0.9.4_1: cannot guess required shlibs, aborting!
phossil in void-packages on  PDFKit [?] took 59s ✗
```